MiscVBAFunctions/Modules/Test__MiscArray.bas

```diff
@@ -244,6 +244,46 @@ TestFail:
     Resume TestExit
 End Sub
 
+'@TestMethod("MiscArray")
+Private Sub Test_ArrayToRange_FunkyHeaders()
+    On Error GoTo TestFail
+    
+    'Arrange:
+    Dim WB As New Workbook
+    Dim Arr(1, 3) As Variant
+    Dim RangeObj As Range
+    Dim RangeOutput As Range
+    
+    'Act:
+    Set WB = ExcelBook("", False, False)
+    Arr(0, 0) = "asdf"
+    Arr(0, 1) = 1234
+    Arr(0, 2) = "2022/11/02"
+    Arr(0, 3) = False
```

Comment:
What would happen with "False"? Stay `False` in the header? :)

Reply:
It becomes the string "FALSE"